### DISCUSSION TITLE
How to change heap size for enabling word suggestion?

### DISCUSSION THREAD

**Discussion by Blade694** [florisboard/florisboard#1316] [Q&A]
I can enable(although a bit laggy) word suggestion by choosing override heap size restriction in the advanced settings. I'm unable to find where i can manually change heap size or keyboard size to enable word suggestion. I'm on Android 11. I tried to change it from the options in smartbar but i can't seem to find anything by which i can change it. I know that that a word suggestion related issue is pinned but there wasn't anything related to it being not able to enable in known issues so I'm asking this. 

Help?

**Comment by patrickgold** [comment]
You can't - the heap size is managed by your device's Java runtime and that decides on itself what process gets how much. In 0.3.15 I will move word suggestions to native memory and this limit will not be relevant anymore. Until then, sadly you either have to live with the word suggestions crashing sometimes or turning them off. Hope you can understand!

**Reply by Blade694** [reply]
Thanks for the heads up. I'll wait for the update. Kudos for the app btw. Stay happy and stay safe. 😀
